client/src/redux/actions/gameStateActions.js

```diff
@@ -1,14 +1,19 @@
+import _ from 'lodash';
+
+import CardInfo from '../../models/cardInfo';
+
 const types = {
   UPDATE_PLAYER: 'UPDATE_PLAYER',
-  UPDATE_ZONE: 'UPDATE_ZONE'
+  UPDATE_ZONE: 'UPDATE_ZONE',
```

Comment:
UPDATE_ZONE won't exist, and there should be a new UPDATE_CARD, not sure how exactly that needs to work. Merging that in with this stuff may be kinda messy... We should probably figure this out in our next meeting

Reply:
After working on my stuff I actually don't think it'll be bad to merge.

Reply:
No worries, I trashed UPDATE_ZONE. The UI will probably use the state.zone property on cards to determine when zones change so I don't think this action will be necessary